OUDS/Core/Components/Sources/Switch/Internal/OUDSSwitchLabel.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Software Name: OUDS iOS
+// SPDX-FileCopyrightText: Copyright (c) Orange SA
+// SPDX-License-Identifier: MIT
+// 
+// This software is distributed under the MIT license,
+// the text of which is available at https://opensource.org/license/MIT/
+// or see the "LICENSE" file for more details.
+// 
+// Authors: See CONTRIBUTORS.txt
+// Software description: A SwiftUI components library with code examples for Orange Unified Design System 
+//
+
+import OUDS
+import OUDSTokensSemantic
+import SwiftUI
+
+struct OUDSSwitchLabel: View {
+
+    // MARK: Stored properties
+
+    @Environment(\.theme) private var theme
+    @Environment(\.colorScheme) private var colorScheme
+
+    struct Label {
+        let label: String
+        let helperText: String?
+        let icon: Image?
+        let onError: Bool
```

Comment:
Je dirais plutôt "isError" que "onError".
C'est un *Bool*, donc "is" est à privilégier par rapport à "on" qui me fait plutôt penser à des closures.